include/openspace/navigation/orbitalnavigator.h

```diff
@@ -112,6 +114,9 @@ class OrbitalNavigator : public properties::PropertyOwner {
     void setRetargetInterpolationTime(float durationInSeconds);
     void updatePreviousStateVariables();
 
+    void setMinimumAllowedDistance(float distance);
+    void setMaximumAllowedDistance(double distance);
```

Comment:
Why is one of them a `float` and the other a `double`?  I'd assume that both should be doubles?

Reply:
The properties are floats so I'm changing both of these to use floats instead

Reply:
No, floats is fine;  was just assuming the wider type